go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0 // indirect
 	github.com/grpc-ecosystem/grpc-gateway v1.5.1 // indirect
 	github.com/jonboulle/clockwork v0.1.0 // indirect
-	github.com/juju/errors v0.0.0-20181012004132-a4583d0a56ea
+	github.com/pingcap/errors v0.11.0
 	github.com/juju/loggo v0.0.0-20180524022052-584905176618 // indirect
```

Comment:
need them?

Reply:
you mean juju/loggo and juju/testing? I will remove them